nominees/coinsence.json

```diff
@@ -1,16 +1,31 @@
 {
   "name": "Coinsence",
-  "description": "Platform to match youth volunteers to social enterprises",
+  "aliases": [
+    ""
+  ],
+  "description": "A global decentral collaboration platform and change-maker network powered by blockchain based impact currencies",
   "website": "https://coinsence.org",
   "license": [
     {
       "spdx": "MIT",
-      "licenseURL": "https://opensource.org/licenses/mit-license.php"
+      "licenseURL": "https://github.com/Coinsence/humhub-modules-xcoin/blob/master/LICENSE"
     }
   ],
   "SDGs": [
     {
-      "SDGNumber": 17
+      "SDGNumber": 10,
+      "evidenceText": "Through alternatives for funding and value sharing based on crypto assets, we are building a new way how communities can share profit within the ecosystem (with contributors and well as with customers).",
+      "evidenceURL": "https://www.unicef.org/innovation/fundgraduate/Coinsence"
+    },
+    {
+      "SDGNumber": 16,
+      "evidenceText": "Through our solution communities are building decentral governance frameworks that create the possibility for people and organization to collaborate e.g. based on democratic consensus. Through creating new digital ways for voting and policy making across different groups, regional and national borders, we can foster peace and justice and build alternatives to some todays non-functioning institutions.",
+      "evidenceURL": "https://www.unicef.org/innovation/fundgraduate/Coinsence"
```

Comment:
Are they aiming for SDG 16.6, "_Develop effective, accountable and transparent institutions at all levels_"? I'm having a hard time connecting this statement to the targets and indicators. And does this take into account only the Xcoin module too?